packages/frontend/amp/components/AdConsent.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { AMPJSON } from './AMPJSON';
+import { css, cx } from 'emotion';
+import Tick from '@guardian/pasteup/icons/tick.svg';
+
+const fontFamily =
+    'GuardianTextSans,Helvetica Neue,Helvetica,Arial,Lucida Grande,sans-serif';
+
+const consentUIStyle = css`
+    font-family: ${fontFamily};
+    font-size: 1rem;
+    line-height: 1.25rem;
```

Comment:
We have a fancy new way of maintaining consistent typography using [`@guardian/pasteup`](https://github.com/guardian/dotcom-rendering/tree/master/packages/pasteup#fonts)

Reply:
I need some precise guidance on that :)

Reply:
For the above, I would probably use:

```js
const consentUIStyle = css`
  ${textSans(3)};
`;
```

Reply:
@SiAdcock Is it at all feasible to lint these kind of things?  I can see this being difficult to spot in a large PR.  (But I can also see building complex linting rules as equally difficult/impossible so just a query!)

Reply:
@kelvin-chappell I totally agree, and it's definitely on our todo list!